ConfigApp/Effects.cs

```diff
@@ -588,6 +589,7 @@ public enum EffectType
             {EffectType.EFFECT_PLAYER_AIMBOT, new EffectInfo("Aimbot", EffectCategory.PLAYER, "player_aimbot", true)},
             {EffectType.EFFECT_PEDS_SPAWN_BIKER, new EffectInfo("Spawn Biker", EffectCategory.PEDS, "peds_spawn_biker")},
             {EffectType.EFFECT_PEDS_SPAWN_SPACE_RANGER, new EffectInfo("Spawn Space Ranger", EffectCategory.PEDS, "peds_spawn_space_ranger")},
+            {EffectType.EFFECT_REPOSSESSION, new EffectInfo("Repossession", EffectCategory.VEHICLE, "veh_repossession", true, true) },
```

Comment:
Any reason for the `IsTimed` and `IsShortTimed` flags here?

Reply:
Yea, I was testing another effect I made locally and I guess I just changed the effect name to "EFFECT_REPOSSESSION" when I saw the request for this. I only replaced the effect name and I guess my dumbo brain forgot to change the flags. I'll fix that now.